lte/gateway/c/li_agent/src/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.7.2)
+
+PROJECT(LIAgent)
+
+find_package(SERVICE303_LIB REQUIRED)
+find_package(SERVICE_REGISTRY REQUIRED)
+find_package(MAGMA_CONFIG REQUIRED)
+find_package(ASYNC_GRPC REQUIRED)
+find_package(MAGMA_LOGGING REQUIRED)
+
+include($ENV{MAGMA_ROOT}/orc8r/gateway/c/common/CMakeProtoMacros.txt)
+
+if (BUILD_TESTS)
+  message("Adding code coverage build and linker flags for BUILD_TESTS")
+  set(CMAKE_CXX_FLAGS_DEBUG
+      "${CMAKE_CXX_FLAGS_DEBUG} -g -O0 -fprofile-arcs -ftest-coverage")
+  set(CMAKE_LINKER_FLAGS_DEBUG
+      "${CMAKE_LINKER_FLAGS_DEBUG} -fprofile-arcs -ftest-coverage")
+endif ()
+
+# Add AddressSanitizer (asan) support for debug builds of LiAgentD
+set(CMAKE_CXX_FLAGS_DEBUG
+    "${CMAKE_CXX_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
+set(CMAKE_LINKER_FLAGS_DEBUG
+    "${CMAKE_LINKER_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
+# Add LeakSanitizer (lsan) support to the release build of LIAgentD so that
+# we can find memory leaks in production.
+set(CMAKE_C_FLAGS_RELWITHDEBINFO
+    "${CMAKE_C_FLAGS_RELWITHDEBINFO} -fsanitize=leak -fno-omit-frame-pointer")
+set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
+    "${CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO} -fsanitize=leak")
+
+add_definitions(-DLOG_WITH_GLOG)
+
+# compile the needed macros
+create_proto_dir("lte" LTE_CPP_OUT_DIR)
```

Comment:
this is unrelated to your change but i think we don't need `spgw_service` for this service? or do we? or policydb :o

Reply:
Let me take a look. It seemed like session_manager added a dependency into policydb.

Reply:
ok so mobilityd client brings in subscriberdb which brings in APN. Thats the minimum set of protos we need. Fixed and updated, thanks for the review.

Reply:
thanks! that was bothering me for a bit :)